plugins/woocommerce/tests/e2e-pw/envs/default-pressable/playwright.config.js

```diff
@@ -6,7 +6,7 @@ config = {
 	retries: 0,
 	projects: [
 		{
-			name: 'default',
+			name: 'default pressable',
```

Comment:
Out of curiosity, why this change?

Reply:
Looks better in console logs, nothing more :)